pom.xml

```diff
@@ -138,6 +138,8 @@
         <geronimo-atinject.version>1.2</geronimo-atinject.version>
         <mongo-java-driver-version>4.11.1</mongo-java-driver-version>
         <velocity-version>2.3</velocity-version>
+        <xmlresolver-bundle-version>5.2.1_1</xmlresolver-bundle-version>
+        <saxon-bundle-version>12.2_1</saxon-bundle-version>
```

Comment:
Warning it is not the version of Saxon used in Camel 4.4 as you can see https://github.com/apache/camel/blob/camel-4.4.x/parent/pom.xml#L415. Make sure to compare the version used in Camel, it will remain our reference